2024/survey.yaml

```diff
@@ -331,7 +330,6 @@ questions:
       - Reddit (unofficial)
       - Wiki (unofficial)
       - Blog posts
-      - Others
```

Comment:
This one may be needed since it's an item in ranking question, not a "other" free text field in a single/multiple choice/s question.

It's your question, it's up to you to keep it or not.

Reply:
Thanks for the catch! Added it back.